scss/_tables.scss

```diff
@@ -4,63 +4,66 @@
   thead {
     // normal header row
     th {
-      border-bottom: solid .125rem theme-color('primary');
       border-top: none;
+      border-bottom: solid .125rem theme-color("primary");
 
       > .material-icons {
-        color: $gray-500;
         margin-top: -.5rem;
+        color: $gray-500;
       }
     }
 
     // filter row
-    tr {
-      &.column-filters {
-        background: $gray-200;
-        th {
-          border-bottom: none;
-          vertical-align: top;
-          padding: {
-            top: 1rem;
-            bottom: 1rem;
-          }
+    .column-filters {
```

Comment:
Why drop the `tr` ?

Reply:
same thing as there : https://stylelint.io/user-guide/rules/selector-no-qualifying-type